content/24.vaccines-app.md

```diff
@@ -1,6 +1,5 @@
-## Vaccine Development Strategies for SARS-CoV-2 Appendix
-
-One of the two mRNA vaccines, Pfizer and BioNTech's BNT162b2, has been issued an EUA for patients as young as 16 [@url:https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-submit-emergency-use-authorization], while ModernaTX has begun a clinical trial to assess its mRNA vaccine in adolescents ages 12 to 18 [@url:https://investors.modernatx.com/news-releases/news-release-details/moderna-announces-first-participants-dosed-phase-23-study-covid].
+## Appendix: Vaccine Development Strategies for SARS-CoV-2
+<!-- It would be ideal to loosely organize these sections in the order they appear in the main text, or even split the appendix into a traditional and novel vaccines appendix. -->
```

Comment:
Done-ish in a new PR that I'll open as soon as I incorporate these edits!

Reply:
Great! Yes, let's save that for a separate pull request.